SETUP_LOCAL_CLUSTER.md

```diff
@@ -0,0 +1,72 @@
+# Introduction
+
+For both HOPR app and core protocol developers, being able to spin a fully interconnected HOPR network is critical.
```

Comment:
```suggestion
The best way to learn and develop the HOPR protocol is by interacting with a HOPR node connected with the HOPR network. Since the HOPR network requires real money to interact, the alternative involves spinning a HOPR cluster locally or within a cloud setup to mimic the same setup.
```